cerbero/utils/__init__.py

```diff
@@ -577,3 +580,16 @@ def is_text_file(filename):
     global TEXTCHARS
     with open(filename, 'rb') as f:
         return len(f.read(1024).translate(None, TEXTCHARS)) <= 10
+
+@lru_cache(maxsize=None)
+def get_class_checksum(clazz):
+    '''
+    Return the SHA256 hash from the source lines of a class.
+    This method uses an LRU cache to avoid calculating the same
```

Comment:
typo I think, but I am not sure: an LRU -> a LRU

Reply:
Well, this is an internal debate I always have. Since you pronounce LRU as /EL AR IU/, shouldn't it be *an* "EL", since the final N is to help break 2 consecutive vowels?

Reply:
Me too I never know what is correct

Reply:
an LRU => it's always based on sound, if it sounds like a vowel it's 'an' else it's 'a'.
Curiously in english Y is not a vowel, so if it sounds like 'i' it's 'a'